qubes/backup.py

```diff
@@ -512,11 +514,15 @@ def _send_progress_update(self):
         if not self.total_backup_bytes:
             return
         if callable(self.progress_callback):
-            progress = (
-                100 * (self._done_vms_bytes + self._current_vm_bytes) /
-                self.total_backup_bytes)
-            # pylint: disable=not-callable
-            self.progress_callback(progress)
+            if time.time() - self.last_progress_time >= 1: # avoid flooding
+                progress = (
+                    100 * (self._done_vms_bytes + self._current_vm_bytes) /
+                    self.total_backup_bytes)
+                if progress - self.last_progress >= 0.5:
```

Comment:
Command line `qvm-backup` displays up to 2 digits. With large backups it makes a difference.
IMO this should be either 0.01, or totally dropped (leaving only time based ratelimit).

Reply:
ok better dropped